### ISSUE TITLE
Website doesn't listen correctly

### ISSUE THREAD

**Issue by ogbaba** [Kitt-AI/snowboy#222]
Hi, this is a simple but annoying issue : When recording a sample using the website in "Record my voice" the recording just doesn't stop and in "Test the model" it just doesn't recognize anything even though I can see it detects my voice…

**Comment by chenguoguo** [comment]
were you using it in a noisy environment? the voice activity detector may not work properly in a noisy environment.

**Comment by ogbaba** [comment]
No, I tried with two microphones (my laptop's integrated mic and an external small microphone), while the external mic had a bit of noise, the other one seemed to work fine with audacity…

**Comment by chenguoguo** [comment]
What browser are you using? If you are using Chrome, could you try to run it in incognito mode? Also, try to log out and log in again and see if that works.

**Comment by ogbaba** [comment]
I am using Mozilla Firefox 45.9.0
